docs/eventlisteners.md

```diff
@@ -85,6 +85,9 @@ To be an Event Interceptor, a Kubernetes object should:
 * Accept JSON payloads over HTTP
 * Return a HTTP 200 OK Status if the EventListener should continue processing
   the event
+* Return HTTP headers back. These will be used by the EventListener as the event
```

Comment:
I think this makes senses, but overall maybe we could consolidate the bullets here?

Reply:
What exactly do you mean?